src/main/webapp/gast/stat.jsp

```diff
@@ -0,0 +1,170 @@
+    }
+
+    public void print_pagination(JspWriter out, int currentPage, int recordsPerPage, String filterTitle, int nOfPages, String sort, String jumpToID) throws Exception {
+        out.println("<nav aria-label=\"Navigation for rows\">");
+        out.println("<ul class=\"pagination\">");
+
+        /*Print Previous Button*/
+        if (currentPage != 1) {
+            out.println(html_prev_button(filterTitle, String.valueOf(recordsPerPage), currentPage,sort,jumpToID));
+        }
+
+        /*Print Pages and highlight current page*/
+        for (int i = 1; i <= nOfPages; i++) {
+            if (currentPage == i) {
+                out.println(html_page_item_current(i));
+            } else {
+                out.println(html_page_item(i, filterTitle, recordsPerPage,sort,jumpToID));
+            }
+        }
+
+        /*Print Next Button*/
+        if (currentPage < nOfPages) {
+            out.println(html_next_button(filterTitle, String.valueOf(recordsPerPage), currentPage,sort,jumpToID));
+        }
+        out.println("</ul>");
+        out.println("</nav>");
+    }
+%>
+<p>
+        
+    <h1>Statistik</h1>
+    <a href="<%=Utils.getBaseUrl(request)%>/gast/quelle">Zurück</a>
+    <br><br>
+    <h3>Liste der Quellen mit Anzahl der Belege
+    </h3>
+    <br>
+    <%
+        print_pagination(out, currentPage, recordsPerPage, filterTitle, nOfPages,sort,jumpToID);
+    %> 
+    <table id="stat1" class="statTable">
+        <thead>        
+            <th>
+                <b>Titel der Quelle</b>
+                <form method="GET">
+                    <input name="filterTitle" type="text" size="40" value="<%=filterTitle %>" placeholder="Titel Filter"/>
+                    <input name="page" type="hidden" value="anzahl_belege"/>
+                    <input name="sort" type="hidden" value="<%=sort %>"/>
+                    <input type="submit" />
+                </form>
+                <%
+                    out.println(html_sort_title_up(currentPage, filterTitle, recordsPerPage,jumpToID));
+                    out.println(html_sort_title_down(currentPage, filterTitle, recordsPerPage,jumpToID));
+                %>
+            </th>
+            <th>
+                <b>Anzahl Belege</b>
+                <br>
+                <%
+                    out.println(html_sort_belege_up(currentPage, filterTitle, recordsPerPage, jumpToID));
+                    out.println(html_sort_belege_down(currentPage, filterTitle, recordsPerPage, jumpToID));
+                %>
+            </th>
+        </thead>
+       <tbody>
+            <%
+            for (Quelle q : lst) {
+                out.print("<tr>");
+                out.print("<td>");
+                out.print("<a href=\""+Utils.getBaseUrl(request)+"/gast/quelle?ID="+String.valueOf(q.getId())+"\">");
+                out.print(Utils.escapeHTML(q.getBezeichnung()));
+                out.print("</a>");
+                out.print("</td>");
+                out.print("<td>");
```

Comment:
The width of the columns in the result table seems to be depending on the length of the entries in the first column depending on the selected page. Can you please give the columns a fixed width in %?

Reply:
i have added a fixed percentage